#VRML_SIM R2020a utf8
WorldInfo {
}
Viewpoint {
  orientation 0.0061719559596435805 -0.9745408165258432 -0.22412519689846336 3.222439243880241
  position 0.052661087678482564 1.3761041872815034 -2.270781145024299
}
TexturedBackground {
}
TexturedBackgroundLight {
}
RectangleArena {
  translation 0 0.01 0
  floorSize 5 5
}
Robot {
  translation 0 0.16 0
  children [
    Solid {
      children [
        Shape {
          appearance Appearance {
            material Material {
              diffuseColor 1 0 0
            }
          }
          geometry Box {
            size 0.1 0.05 0.2
          }
        }
      ]
      name "body"
      boundingObject Box {
        size 0.1 0.05 0.2
      }
      physics Physics {
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        anchor 0.06 0 0.05
      }
      device [
        RotationalMotor {
          name "wheel FL"
        }
      ]
      endPoint Solid {
        translation 0.06 0 0.05
        rotation 0 0 1 1.5708
        children [
          Shape {
            appearance Appearance {
              material Material {
                diffuseColor 0 1 0
              }
            }
            geometry Cylinder {
              height 0.02
              radius 0.04
            }
          }
        ]
        name "wheel FL"
        boundingObject Cylinder {
          height 0.02
          radius 0.04
        }
        physics Physics {
        }
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        anchor 0.06 0 -0.05
      }
      device [
        RotationalMotor {
          name "wheel BL"
        }
      ]
      endPoint Solid {
        translation 0.06 0 -0.05
        rotation 0 0 1 1.5708
        children [
          Shape {
            appearance Appearance {
              material Material {
                diffuseColor 0 1 0
              }
            }
            geometry Cylinder {
              height 0.02
              radius 0.04
            }
          }
        ]
        name "wheel BL"
        boundingObject Cylinder {
          height 0.02
          radius 0.04
        }
        physics Physics {
        }
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        anchor -0.06 0 0.05
      }
      device [
        RotationalMotor {
          name "wheel FR"
        }
      ]
      endPoint Solid {
        translation -0.06 0 0.05
        rotation 0 0 1 1.5708
        children [
          Shape {
            appearance Appearance {
              material Material {
                diffuseColor 0 1 0
              }
            }
            geometry Cylinder {
              height 0.02
              radius 0.04
            }
          }
        ]
        name "wheel FR"
        boundingObject Cylinder {
          height 0.02
          radius 0.04
        }
        physics Physics {
        }
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        anchor -0.06 0 -0.05
      }
      device [
        RotationalMotor {
          name "wheel BR"
        }
      ]
      endPoint Solid {
        translation -0.06 0 -0.05
        rotation 0 0 1 1.5708
        children [
          Shape {
            appearance Appearance {
              material Material {
                diffuseColor 0 1 0
              }
            }
            geometry Cylinder {
              height 0.02
              radius 0.04
            }
          }
        ]
        name "wheel BR"
        boundingObject Cylinder {
          height 0.02
          radius 0.04
        }
        physics Physics {
        }
      }
    }
  ]
  boundingObject Box {
    size 0.1 0.05 0.2
  }
  physics Physics {
  }
  controller "<extern>"
}
